public/app/features/dashboard-scene/scene/DashboardControls.tsx

```diff
@@ -122,10 +122,10 @@ function DashboardControlsRenderer({ model }: SceneComponentProps<DashboardContr
   const { variableControls, refreshPicker, timePicker, hideTimeControls, hideVariableControls, hideLinksControls } =
     model.useState();
   const dashboard = getDashboardSceneFor(model);
-  const { links, editPanel, $timeRange } = dashboard.useState();
+  const { links, editPanel } = dashboard.useState();
   const styles = useStyles2(getStyles);
   const showDebugger = location.search.includes('scene-debugger');
-  $timeRange!.useState();
+  sceneGraph.getTimeRange(dashboard).useState();
```

Comment:
why is this not done inside DashboardLinksControls? looks like there is no need to re-render on this level on time range change

Reply:
indeed, there is no need for it in the controls. I moved it back in link controls, moved the tests there as well.